.github/kokoro/db-full.sh

```diff
@@ -54,18 +54,18 @@ echo "----------------------------------------"
 	# which is well before the 05x INT fuzzers complete.
 	export MAX_VIVADO_PROCESS=$((CORES/2 < 20 ? CORES/2 : 20))
 	set -x +e
-	script --return --flush --command "make -j $CORES MAX_VIVADO_PROCESS=$MAX_VIVADO_PROCESS" -
+	tmp=`mktemp`
+	script --return --flush --command "make -j $CORES MAX_VIVADO_PROCESS=$MAX_VIVADO_PROCESS" $tmp
```

Comment:
As far as I can tell, this printed to STDOUT before.  Should you be using "tee" here?

Reply:
It still is printed to STDOUT.
From script manual:
> (..)script  makes a typescript of everything displayed on your terminal. (...) If the argument file is given, script saves the dialogue in this file.  If no filename is given, the dialogue is saved in the file typescript.

I just added the temp file to easily delete it after is it used.

Reply:
This doesn't change the fact that there is a bug in where I delete the temp file. It fails because it is in a different scope. I will fix that right away.